public/icons/arrowRight.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import { SVGProps } from "react";
+
+export const ArrowRight = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
La size/colore dell'icona credo sia più comodo averle definite all'esterno, in questo modo se in un punto dell'app hai bisogno della stessa icona ma più grande o di un altro colore riesci a farlo senza problemi

Disclaimer: non so quanto sia utile ai fini del webinar

Reply:
Mi torna! E penso abbia senso strutturarlo in questo modo, lo correggo come mi hai segnalato 🙌🏻